pkg/actors/actors.go

```diff
@@ -1042,6 +1046,20 @@ func (a *actorsRuntime) CreateTimer(ctx context.Context, req *CreateTimerRequest
 			}
 		}
 
+		// Check if timer is still active
+		select {
+		case <-stop:
+			log.Infof("Time: %v with parameters: DueTime: %v, Period: %v, Data: %v has been deleted.", timerKey, req.DueTime, req.Period, req.Data)
+			return
+		default:
+			break
+		}
+
+		err := a.executeTimer(actorType, actorID, name, dueTime, period, callback, data)
```

Comment:
Why do we need call executeTimer here?

Reply:
Else, the timer is not firing after dueTime...it fires only after the period interval is over

Reply:
Sorry for the late. How did it work before?

Reply:
I dont think it worked